frontend/src/components/SearchBar/styles.scss

```diff
@@ -0,0 +1,19 @@
+$bg-color: #ffffff;
+$border-color: #A61700;
+$big-border-radius: 50px;
```

Comment:
Ideally, you can define variables.scss and import it everywhere needed, contact with @Uzhastin-Nikita to resolve it